creusot-contracts/src/std.rs

```diff
@@ -13,6 +13,7 @@ pub mod num;
 pub mod ops;
 pub mod option;
 pub mod panicking;
+pub mod pointer;
```

Comment:
```suggestion
pub mod ptr;
```

In the `std` module, we try to follow Rust's stdlib structure.

Reply:
Also, we re-export the corresponding std module in each of these, so that the user does not have to import both (I know this has not been done everywhere, but this should be fixed.).

Reply:
done, thanks